mongo/integration/client_side_encryption_prose_test.go

```diff
@@ -1784,3 +1812,12 @@ func (d *deadlockTest) disconnect(mt *mtest.T) {
 	d.clientTest.Disconnect(context.Background())
 	assert.Nil(mt, err, "clientTest Disconnect error: %v", err)
 }
+
+// getCSFLEExtraOptions returns an AutoEncryption extra options map with csfle required set and a
+// csfle library override path set if available.
+func getCSFLEExtraOptions() map[string]interface{} {
+	return map[string]interface{}{
```

Comment:
That seems reasonable. The main reason I added those "csfleRequired=true" configurations is to prevent the tests from silently failing over to mongocryptd if the `CSFLE_PATH` env var doesn't get set correctly. However, I can just add that check to `run-tests.sh` so that devs can run all these tests with or without csfle on their workstation.